src/core.jl

```diff
@@ -427,6 +427,31 @@ function Base.map(f, As::AxisArray{T,N,D,Ax}...) where {T,N,D,Ax<:Tuple{Vararg{A
     return AxisArray(map(f, data...), As[1].axes...)
 end
 
+function Base.mapslices(f, A::AxisArray; dims)
+    new_axes = Axis[axes(A)...]
```

Comment:
needs to be splatted into an array so the elements can be mutated

Reply:
```suggestion
    new_axes = collect(axes(A))
```